tests/integration/test_subscriptions_cli.py

```diff
@@ -285,22 +285,26 @@ def test_request_base_success(invoke, geom_geojson):
     assert result.exit_code == 0  # success.
 
 
-def test_request_catalog_success(invoke, geom_geojson):
+@pytest.mark.parametrize('item_types, asset_types',
```

Comment:
I'd recommend leaving this out as it doesn't seem to be a necessary change to the test and adds complexity to the review.